src/test/java/org/jitsi/meet/test/base/AbstractBaseTest.java

```diff
@@ -134,17 +134,7 @@ final public void setupClassPrivate(ITestContext context)
 
         participants = new ParticipantHelper(new ParticipantFactory(config));
 
-        // make sure if setup fails we will cleanup
-        // when configure method @BeforeClass fails @AfterClass is not executed
-        try
-        {
-            setupClass();
-        }
-        catch (Throwable t)
-        {
-            cleanupClass();
-            throw t;
-        }
+        setupClass();
```

Comment:
It will be useful if we had a comment either here or in cleanupClass description that it will be called by FailureListener if that fails.